components/home/sections/newsletter.js

```diff
@@ -1,39 +1,58 @@
 "use client";
-
 import { useState } from "react";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```

Comment:
not really necessary to keep calling import "react-toastify/dist/ReactToastify.css"; on every usecase....can be configured to apply styles globally...

```suggestion
 
```

Reply:
Yah, i think it's not really necessary. Thank you @HassanBahati